modules/passkey/test/4337/userStories/createSafeWithPasskeySignerAsOwner.ts

```diff
@@ -0,0 +1,141 @@
+import { expect } from 'chai'
+import { deployments, ethers, network } from 'hardhat'
+import { packGasParameters } from '@safe-global/safe-4337/dist/src/utils/userOp'
+import { WebAuthnCredentials, decodePublicKey, encodeWebAuthnSignature } from '../../utils/webauthn'
+import { Safe4337Module } from '@safe-global/safe-4337/typechain-types/contracts/Safe4337Module'
+
+describe.only('User story [@4337]', () => {
```

Comment:
I think using the bundler just complicates things. I would just execute a transaction directly over the `EntryPoint` contract - we have other tests that verify the passkey works over 4337 bundlers.